generator-mediator-js/app/prompts.js

```diff
@@ -0,0 +1,112 @@
+'use strict'
+
+var config = require('./config')
+
+exports.promptsList = (enablePrompts) => {
+  var list = [
+    {
+      when: function() { return enablePrompts },
+      type: 'confirm',
+      name: 'enablePrompts',
+      message: 'Would you like to be prompted for config?',
+      default: config.enablePrompts
+    }, {
+      when: function(response) { return response.enablePrompts },
+      type: 'input', 
+      name: 'mediatorName',
+      message: 'What is your Mediator\'s name?',
+      default: config.mediatorName
+    }, {
+      when: function(response) { return response.enablePrompts },
+      type: 'input', 
+      name: 'mediatorDesc', 
+      message: 'What does your Mediator do?',
+      default: config.mediatorDesc
+    }, {
+      when: function(response) { return response.enablePrompts },
+      type: 'input', 
+      name: 'mediatorAuthor', 
+      message: 'Who is the author of this mediator?',
+      default: config.mediatorAuthor
+    }, {
+      when: function(response) { return response.enablePrompts },
+      type: 'input', 
+      name: 'mediatorMaintainer', 
+      message: 'Who is the maintainer of the mediator?',
+      default: config.mediatorMaintainer
+    }, {
+      when: function(response) { return response.enablePrompts },
+      type: 'input', 
+      name: 'configPort', 
+      message: 'Under what port number should the mediator run?', 
+      default: config.configPort 
+    }, {
+      when: function(response) { return response.enablePrompts },
+      type: 'input', 
+      name: 'defaultChannelPath', 
+      message: 'What is your default channel path?',
+      default: config.defaultChannelPath
+    }, {
+      when: function(response) { return response.enablePrompts },
+      type: 'input', 
+      name: 'mediatorRouteHost', 
+      message: 'What is your mediator ip address?',
```

Comment:
This is confusing, I think should explain what they does. I'm guessing this is used for the default channel and endpoint during OpenHIM registration. I think we should say something like `Will this mediator be run alongside the OpenHIM? IF so, use localhost otherwise specify a hostname for the OpenHIM to forward requests to this mediator` Or something similar to this.